dijit/1.11/dijit.d.ts

```diff
@@ -225,9 +219,7 @@ declare namespace dijit {
 	}
 
 	/* dijit/_FocusMixin */
-	interface _FocusMixin { }
-
-	interface _WidgetBase extends dojo.Stateful, Destroyable {
+	interface _FocusMixin extends _WidgetBase {
```

Comment:
I think we should add the `focused` property to the definition https://github.com/dojo/dijit/blob/master/_FocusMixin.js#L17